test/tests/functional/pbs_cgroups_hook.py

```diff
@@ -1599,23 +1936,23 @@ def tearDown(self):
                                              uid='root', gid='root',
                                              mode=0644)
                             self.du.rm(hostname=self.momA, path=fn)
-                            cmd = ["rmdir", jpath]
-                            self.logger.info("deleting jobdir %s" % cmd)
+                            cmd = 'rmdir ' + jpath
+                            self.logger.info('deleting jobdir %s' % cmd)
                             self.du.run_cmd(cmd=cmd, sudo=True)
         # Remove the jobdir if any under other cgroups
         cgroup_subsys = ('cpuset', 'memory')
         for subsys in cgroup_subsys:
-            self.logger.info("Looking for orphaned jobdir in %s" % subsys)
+            self.logger.info('Looking for orphaned jobdir in %s' % subsys)
             if self.paths[subsys]:
-                cdir = self.paths['cpuset']
+                cdir = self.paths[subsys]
                 if os.path.isdir(cdir):
                     cpath = os.path.join(cdir, 'pbspro')
+                    if not os.path.isdir(cpath):
+                        cpath = os.path.join(cdir, 'pbspro.slice')
                     if os.path.isdir(cpath):
-                        jid = glob(os.path.join(cpath, "*", ""))
-                        if jid:
-                            for files in jid:
-                                self.logger.info("***Found jobid %s" % files)
-                                jdir = os.path.join(cpath, files)
-                                cmd2 = ["rmdir", jdir]
-                                self.logger.info("deleting jobdir %s" % cmd2)
-                                self.du.run_cmd(cmd=cmd2, sudo=True)
+                        for jdir in glob.glob(os.path.join(cpath, '*', '')):
+                            if not os.path.isdir(cpath):
+                                continue
```

Comment:
You know cpath is a dir, you're in a if os.path.isdir(cpath)

Reply:
That should have been jdir and not cpath. Fixed.